### PR TITLE
Update getting started for VS in F# docs

### PR DESCRIPTION
**Pull request by cartermp**
Updates for latest 15.5 bits.

### CODE REVIEW

File: docs/fsharp/get-started/get-started-visual-studio.md [hunk 1]
```diff
@@ -24,14 +24,15 @@ You'll next see a list of Workloads. You can install F# through any of the follo
...
 |--------|------|
-|.NET desktop development|Select **F# language support** from the right-hand side|
-|ASP.NET and web development|Select **F# language support** from the right-hand side|
-|Data storage and processing|Select **F# language support** from the right-hand side|
-|Mobile development with .NET|No action - F# is installed by default|
+| .NET Core cross-platform development | No action - F# is installed by default |
+| ASP.NET and web development | No action - F# is installed by default |
+| Azure development | No action - F# is installed by default |
+| Mobile development with .NET |No action - F# is installed by default|
+| Data science and analytical applications | None. F# is installed by default |
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-visual-studio.md]: it would be good to be consistent:
image

**Review by mairaw — APPROVED** [review]
LGTM. It would only be nice if the No action column had consistent values

File: docs/fsharp/get-started/get-started-visual-studio.md (hunk 1 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-visual-studio.md]: Done it!

**Comment by mairaw** [comment]
CLA was stuck.

**Pull request merged**